### PR TITLE
release: jdk8u362-b09

### PR DESCRIPTION
**Pull request by steelhead31**
shipped:
x64 linux
aarch64 linux
ppc64le
arm32
alpine

### CODE REVIEW

File: linux/jdk/redhat/src/main/packaging/temurin/8/temurin-8-jdk.spec [hunk 1]
```diff
@@ -1,10 +1,10 @@
-%global upstream_version 8u352-b08
+%global upstream_version 8u362-b09
 # Only [A-Za-z0-9.] allowed in version:
 # https://docs.fedoraproject.org/en-US/packaging-guidelines/Versioning/#_upstream_uses_invalid_characters_in_the_version
 # also not very intuitive:
 #  $ rpmdev-vercmp 8.0.312.0.1___8 8.0.312.0.0+7
 #  8.0.312.0.0___7 == 8.0.312.0.0+7
-%global spec_version 8.0.352.0.0.8
+%global spec_version 8.0.362.0.0.9
 %global spec_release 2
```
Comment (zdtsw) [inline linux/jdk/redhat/src/main/packaging/temurin/8/temurin-8-jdk.spec:8]: this need to be set to 1
if it is the first release on this version

Comment (zdtsw) [reply linux/jdk/redhat/src/main/packaging/temurin/8/temurin-8-jdk.spec:8]: lin50 %global upstream_version 8u352-b08-aarch32-20221020 need update for arm32

Comment (zdtsw) [reply linux/jdk/redhat/src/main/packaging/temurin/8/temurin-8-jdk.spec:8]: same as for suse

Comment (steelhead31) [reply linux/jdk/redhat/src/main/packaging/temurin/8/temurin-8-jdk.spec]: all sorted.

**Review by zdtsw — APPROVED** [review]

**Review by gdams — APPROVED** [review]

**Review by jerboaa — APPROVED** [review]

**Pull request merged**